src/Sulu/Bundle/WebsiteBundle/Cache/CacheClearer.php

```diff
@@ -78,7 +79,7 @@ public function clear()
 
             $this->cacheManager->invalidateDomain($request->getHost());
 
-            $this->eventDispatcher->dispatch(Events::CACHE_CLEAR);
+            $this->eventDispatcher->dispatch(new CacheClearEvent(), Events::CACHE_CLEAR);
```

Comment:
I thought they changed this call to only have on parameter? :thinking:

Reply:
And how is this related to this PR anyway?

Reply:
Oh sure I wanted to move this into an own PR 🙈 . 
No it are still 2 parameters the second one the string is optional and needed to be keeped for backward compatibility.

Reply:
Own PR created #5247